TestFoundation/TestNSCompoundPredicate.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  TestNSCompoundPredicate.swift
+//  Foundation
+//
+//  Created by Kevin Lundberg on 12/11/15.
+//  Copyright © 2015 Apple. All rights reserved.
+//
```

Comment:
You should look at another Swift file in this project and add the same legal notice.